airflow/cli/commands/worker_command.py

```diff
@@ -76,17 +85,19 @@ def worker(args):
             stderr=stderr,
         )
         with ctx:
-            sub_proc = subprocess.Popen(['airflow', 'serve_logs'], env=env, close_fds=True)
+            if skip_serve_logs is False:
+                sub_proc = subprocess.Popen(['airflow', 'serve_logs'], env=env, close_fds=True)
             worker.run(**options)
-            sub_proc.kill()
```

Comment:
Don't you need to keep this line and have an "if skip_serve_logs is False:" line above it?

Reply:
refactored